demos/ns-inputter--email/index.html

```diff
@@ -0,0 +1,14 @@
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+</head>
+
+<body>
+  <main class="ndsn">
+    <ns-inputter mask="" separator="" validation="[&quot;isRequired&quot;,&quot;isEmail&quot;]" value="" helper="" name="your-email">
```

Comment:
Attributes `mask` and `separator` are not available for email.
```suggestion
    <ns-inputter validation="[&quot;isRequired&quot;,&quot;isEmail&quot;]" value="" helper="" name="your-email">
```